Logger.Tests/LoggerFixture.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Logger.Tests
+{
+    public class LoggerFixture
+    {
+        private readonly ITxsLogger l = new TxsLogger();
+        public string PropA { get; set; }
+        public string PropB { get; set; }
+
+        [Fact]
+        public void Testing_for_apilogger()
```

Comment:
Test name should be like this  APILog_Should_Be_Write_Log_In_File

Reply:
done